azure_jumpstart_ag/artifacts/AgLogonScript.ps1

```diff
@@ -151,6 +152,52 @@ else {
     Write-Host "ERROR: You have to fork the jumpstart-agora-apps repository!" -ForegroundColor Red
 }
 
+<# THIS CODE IS TEMPORARY COMMENTED DUE TO IMPORT ISSUES WITH SPN.
+#####################################################################
+# Import dashboard reports into Azure Data Explorer
+#####################################################################
+# Get Azure Data Explorer URI
+$adxEndPoint = (az kusto cluster show --name $adxClusterName --resource-group $resourceGroup --query "uri" -o tsv)
+
+# Get access token to make REST API call to Azure Data Exploer Dashabord API. Replace double quotes surrounded with acces token
+$token = (az account get-access-token --scope "https://rtd-metadata.azurewebsites.net/user_impersonation openid profile offline_access" --query "accessToken") -replace "`"", ""
+
+# Prepare authorization header with access token
+$httpHeaders = @{"Authorization" = "Bearer $token"; "Content-Type" = "application/json" }
+
+# Make REST API call to the dashbord endpoint.
```

Comment:
This looks like it's just setting the endpoint, not making a REST API call

Reply:
Yes, due to issues with using SPN to import dashboards. Will use REST API, once we have a solution. Currently this works with user token.